pkgs/development/quickemu/default.nix

```diff
@@ -2,6 +2,7 @@
 , fetchFromGitHub
 , stdenv
 , makeWrapper
+, edk2
```

Comment:
As per my comment regarding additional changes, the package is a requirement of quickemu since commit aeb2a64 (https://github.com/quickemu-project/quickemu/commit/aeb2a64720fd3bfdbb0553a3b3b1249c462b06d9).

In regards of use, I am not sure if you are referring to runtime usage by quickemu or the nix expression. This is my first attempt to update a nix expression.

My first draft of this PR had the edk2 listed within 'runtimePaths' but ofborg complained about it being an unknown string.

Finding documentation on 'runtimePaths' via Google turned out to be fruitless, apologies if this created additional work to review.

Reply:
> My first draft of this PR had the edk2 listed within 'runtimePaths' but ofborg complained about it being an unknown string.

Can you try keeping `edk2` in args (which you have already done currently) then add `edk2` to `runtimePaths`, then build and test the package and see if things works?

Reply:
Thanks for the support @SuperSandro2000 and @bobby285271 

I am cancelling this pull request in favor of https://github.com/NixOS/nixpkgs/pull/149409 which solved the OVMF Firmware issue I just encountered.